root/lib/radxa-otgutils/radxa-otgutils.sh

```diff
@@ -70,6 +70,10 @@ start()
     esac
 
     local UDC=$(ls /sys/class/udc/ | head -1)
+    if [ -z "$UDC" ] ; then
+        echo "UDC is NULL"
```

Comment:
`No USB Device Controller was detected. Did you set your OTG port to host mode?`